gdc_ng_models/models/entity_set.py

```diff
@@ -23,6 +23,23 @@
 from sqlalchemy.dialects import postgresql
 from sqlalchemy.ext import declarative
 from gdc_ng_models.models import accessed, audit
+import enum
```

Comment:
nit: seems like isort is not on for this project?
(otherwise, this would have been placed way at the top of all imports)

Reply:
I added in isort to the pre-commit config and ran it on the one file.

Reply:
I introduced isort, but that breaks a bit of policy of reformatting code when also making code changes.